{{cookiecutter.project_slug}}/production.yml

```diff
@@ -38,7 +38,7 @@ services:
     depends_on:
       - django
     volumes:
-      - production_traefik:/etc/traefik/acme
+      - production_traefik:/etc/traefik/acme:Z
```

Comment:
Initially, I wasn't able to obtain a certificate from Let's Encrypt, it was showing up as self-signed cert from Traefik. 

I've changed this into `z` and it worked so I thought it was wrong... However, I later tried again with `Z` and it worked. I think my problem might be unrelated...

Reply:
If there's another container that needs to access the volume, it should be `:z` so that the volume has a shared label.

If, as is the case with traefik FWIU, only the one container needs to access the volume, it should be `:Z` so that the volume has a private label (which is unique to the container). For a multi-node setup, in order to share the cert with every load balancer container, the acme volume would need to be `:z` so that the volume has a shared labe.

Reply:
I changed the label to `:z` so that you could run multiple traefik containers that share the acme config.
While this is more flexible, it is less secure.